examples/reinforcement_learning/diffusion_policy.py

```diff
@@ -0,0 +1,201 @@
+import numpy as np
+import numpy.core.multiarray as multiarray
+import torch
+import torch.nn as nn
+from huggingface_hub import hf_hub_download
+from torch.serialization import add_safe_globals
+
+from diffusers import DDPMScheduler, UNet1DModel
+
+
+add_safe_globals(
```

Comment:
Why do we need it?

Reply:
After setting weights_only=True (from False), an error occurs for any pretrained model unless we use add_safe_globals to make custom or third-party methods available (since weights_only=True skips the configuration loading). I believe it is a preventative measure by HuggingFace for security reasons, because it explicitly states that if we use weights_only=False, we must trust the authors of the model

Reply:
It is happening at `torch.load()` so, I don't think it has anything to do with Hugging Face. Which torch version are you using?

Reply:
I see, thank you - I am using 2.5.1